synapse/lib/layer.py

```diff
@@ -433,6 +465,8 @@ class StorTypeUtf8(StorType):
 
     def __init__(self, layr):
         StorType.__init__(self, layr, STOR_TYPE_UTF8)
+        self.indxIsInvertible = False
```

Comment:
If evaluation of each index value for it's invertibility is feasible, we may benefit from it due to most instances of utf8 index values being invertible.

Reply:
You could just make the decodeIndx API fall back to retrieving the value from the layer rather than making the distinction at all.

Reply:
This case is complicated. It is only invertible if it is <= 256 bytes.